index.html

```diff
@@ -0,0 +1,186 @@
+            </p>
+            <ul class="content-laguages">
+               <li class="lang"><a href="#">html</a></li>
+               <li class="lang"><a href="#">css</a></li>
+               <li class="lang"><a href="#">javascript</a></li>
+            </ul>
+            <div>
+                <button type="submit" class="project-btn">See Project</button>
+            </div>
+        </div>           
+        <div class="work-items">
+           <a class="design" href="#"> <img src="./images/Avialability.png" alt="Availability"></a>
+             <h1 class="work-title">Multi-Post 
+                        <br>Stories</h1>
+             <ul class="projects">
+               <li class="work-name"><a href="#">CANOPY</a></li>
+               <li class="counter"><img src="./images/Counter.png"></li>
+               <li><a class="backend-dev">Back End Dev</a></li>
+               <li class="counter"><img src="./images/Counter.png"></li>
+               <li><a class="backend-dev" href="#">2015</a></li>
+             </ul>
+             <p class="content-work"> 
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.  
+             </p>
+             <ul class="content-laguages">
+                <li class="lang">html</a></li>
+                <li class="lang">css</a></li>
+                <li class="lang">javascript</a></li>
+             </ul>
+             <div>
+             <button type="submit" class="project-btn">See Project</button>
+             </div>
+        </div>
+        <div class="work-items">
+             <a class="design" href="#"> <img src="./images/gymfit.png" alt="Gymfit"></a>
+             <h1 class="work-title">Tonic </h1>
+             <ul class="projects">
+                 <li class="work-name"><a href="#">CANOPY</a></li>
+                 <li class="counter"><img src="./images/Counter.png"></li>
+                 <li><a class="backend-dev">Back End Dev</a></li>
+                 <li class="counter"><img src="./images/Counter.png"></li>
+                 <li><a class="backend-dev" href="#">2015</a></li>
+              </ul>
+             <p class="content-work"> 
+                 A daily selection of privately personalized reads; no accounts or sign-ups required.  
+             </p>
+
+             <ul class="content-laguages">
+                <li class="lang"><a href="#">html</a></li>
+                <li class="lang"><a href="#">css</a></li>
+                <li class="lang"><a href="#">javascript</a></li>
+             </ul>
+             <div>
+                  <button type="submit" class="project-btn">See Project</button>
+             </div>
+        </div>
+        <div class="work-items">
+              <a class="design" href="#"> <img src="./images/art-printing.png" alt="Art printing"></a>
+                  
+              <h1 class="work-title">Multi-Post
```

Comment:
:red_circle:  Please make sure you change this header tag to any other header tag (h2,h3 ...); As requested by the previous reviewer your HTML should not have more than one  `<h1> ` tag.  H1 on the page indicates what the entire web page is about. And the rest of the headings follow a hierarchical structure, with H2 for each topic section and any subsections to those being indicated with H3 and if the subtopic has subtopics then going down as low as H4.

I think this code should be `<h3 class="work-title">Multi-Post <br>Stories </h3>`